virttest/utils_net.py

```diff
@@ -4779,17 +4783,16 @@ def delete_linux_bridge_tmux(linux_bridge_name, iface_name=None, ignore_status=F
     """
     # Delete the linux bridge
     br_path = "/sys/class/net/%s" % linux_bridge_name
-    if not utils_package.package_install(
-        ["tmux", "dhcp-client", "procps-ng", "net-tools"]
-    ):
+    if not utils_package.package_install(["tmux", "dhcpcd", "procps-ng", "net-tools"]):
         raise exceptions.TestError("Failed to install the required packages.")
     if not os.path.exists(br_path):
         LOG.info("There is no bridge named '%s' on the host" % linux_bridge_name)
         return
+    libvirt_service.ensure_service_status("dhcpcd")
     if iface_name:
         cmd = (
-            'tmux -c "ip link set {1} nomaster; ip link delete {0}; pkill dhclient; '
-            'sleep 5; dhclient {1}"'.format(linux_bridge_name, iface_name)
+            'tmux -c "ip link set {1} nomaster; ip link delete {0}; dhcpcd -k {1}; '
```

Comment:
”dhcpcd -k“ could be better, release process for all interfaces.

Reply:
After discussion and research, we found if without $interface specified, it needs dhcpcd.service by manually started. And "dhcpcd -k" makes the serivce stopped, so need restart it again.
Furthermore, "dhcpcd $interface" can do simultaneous operation on different interfaces without the " already running - exiting" obstacles as "dhclient" cmd.
So using dhcpcd with $interface specified is a good usage choice. Let's keep it.
Thanks haijiao.